core/block_files/blockchain.py

```diff
@@ -74,6 +76,53 @@ def proof_of_work(self, block: NimlothBlock) -> str:
             computed_hash = block.compute_hash()
         return computed_hash
 
+    # TODO:add new block creation and new proof of work
+    def proof_of_work_dev(self):
+        return None
+
+    def get_noonce(self, new_block: NimlothBlock) -> int:
+        new_block_hash = new_block.hash
+        noonce = new_block.nonce
```

Comment:
why is it `get_noonce` instead of `get_nonce`, same with variable names

Reply:
because i cant spell